src/SFA.DAS.ApprenticeFeedback.Database/StoredProcedures/GenerateFeedbackSummaries.sql

```diff
@@ -79,7 +80,7 @@ BEGIN
               ,SUM(AttributeValue) Agree 
               ,SUM(CASE WHEN AttributeValue = 1 THEN 0 ELSE 1 END) Disagree
         FROM (
-            SELECT AttributeId, AttributeValue, Ukprn, TimePeriod, COUNT(*) OVER (PARTITION BY TimePeriod, Ukprn, AttributeId) ReviewCount
+            SELECT AttributeId, AttributeValue, Ukprn, TimePeriod, COUNT(*) OVER (PARTITION BY TimePeriod, Ukprn)/@attributes ReviewCount

```

Comment:
This does work now but this will introduce a subtle bug where the number of attributes changes in a feedback during an academic year - which just happened - so it does happen! - or the whole table is recalculated for some reason when there 12 in some and 13 say in other.

If the Latest results also included ApprenticeFeedbackResultId - then the Review Count could be calculated as:
COUNT(*) OVER (PARTITION BY TimePeriod, Ukprn) / COUNT(*) OVER (PARTITION BY TimePeriod, Ukprn, ApprenticeFeedbackResultId) ReviewCount

And this should then calculate the count based on the actual number of attributes in a given feedback

Reply:
This was intended as a quick fix to resolve a prod issue.
As noted in the bug ticket a ore complete solution is required, as the two sets of questions (not) showing in FAT also needs to be tested.
... although I agree this suggestion looks like it will work.